app/javascript/controllers/index.js

```diff
@@ -39,6 +41,7 @@ export const controllerDefinitions = [
   [TypeaheadListController, './refine/typeahead-list-controller.js'],
   [UpdateController, 'refine/update-controller.js'],
   [DateController, 'refine/date-controller.js'],
+  [inlineAdvancedModalController, 'refine/inline-advanced-modal-controller.js'],
```

Comment:
the `InlineAdvancedModalController` looks to be spelled 2 different ways here

Reply:
Good catch! Surprised the code even compiled still from that 🤔